Signal/src/util/OWSScreenLockUI.m

```diff
@@ -534,6 +520,7 @@ - (void)updateScreenBlockingWindow:(ScreenLockUIState)desiredUIState animated:(B
         // Show the blocking window in front of the status bar.
         self.screenBlockingWindow.windowLevel = UIWindowLevelStatusBar + 1;
         self.rootWindow.hidden = YES;
+        [self.screenBlockingViewController becomeFirstResponder];
```

Comment:
is the screenblockingViewController capable of becoming first responder at this time? e.g. I don't think a not-yet presented view controller can become first responder.

Reply:
We could just add an assert that verifies this does what you're intending.

Reply:
Done.